types/mytime.go

```diff
@@ -25,7 +25,7 @@ type MysqlTime struct {
 	month uint8  // month <= 12
 	day   uint8  // day <= 31
 	// When it's type is Time, HH:MM:SS may be 839:59:59, so use int to avoid overflow.
-	hour        int   // hour <= 23
+	hour        int16 // hour <= 23
```

Comment:
Do we need to reorder these fields to decrease its memory consumption after memory alignment?

Reply:
It seems that it's already the best definition order for Time and MysqlTime.
``` go
unsafe.Sizeof(Time{})      // 16
unsafe.Sizeof(MysqlTime{}) // 12
```

Reply:
nice